api/models/rules.py

```diff
@@ -0,0 +1,28 @@
+"""Rule model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .year import Year
+
+
+class Rules(Base):
+    """Stores rules, terms and conditions for year."""
+
+    year = models.ForeignKey(
+        Year,
+        verbose_name=_("Year"),
+    )
+    text = models.TextField(
```

Comment:
I don't agree. The preview, checking and other features would be worth the effort for me.

But I already approved and you are free to merge. You will be the admin anyway.